README.md

```diff
@@ -142,6 +156,43 @@ Name | Description | Type | Default | Required |
 
 See [Waggle Dance README](https://github.com/HotelsDotCom/waggle-dance/README.md) for more information on all these parameters.
 
+### remote_region_metastores
+
+A list of maps.  Each map entry describes a federated metastore endpoint accessible via an AWS VPC endpoint.
```

Comment:
If this list is only for Alluxio, then it should say so here in the description:
```suggestion
A list of maps.  Each map entry describes a federated metastore endpoint accessible via an AWS VPC endpoint. The actual data for these metastores will be accessed using Alluxio caching instead of reading the data from S3 directly.
```

The more I look at this PR, though, the more I think `remote_region_metastores` is completely independent of which metastores in WD will/can do Alluxio filtering.  So maybe ignore what I said above in this comment, but somewhere a more concrete example should be given.